com.unity.render-pipelines.universal/Editor/Converter/RenderPipelineConvertersEditor.cs

```diff
@@ -117,17 +120,36 @@ internal static void DontSaveToLayout(EditorWindow wnd)
 
         void OnEnable()
         {
-            m_CoreConvertersList = new List<RenderPipelineConverter>();
-
             // This is the drop down choices.
             m_ConverterContainers = TypeCache.GetTypesDerivedFrom<RenderPipelineConverterContainer>();
+            foreach (var cont in m_ConverterContainers)
+            {
+                var container = (RenderPipelineConverterContainer)Activator.CreateInstance(cont);
+                m_Containers.Add(container);
+                m_ContainerChoices.Add(container.name);
+            }
+
```

Comment:
Should `m_ContainerChoiceIndex` be reset to 0? All other code below assumes` m_ContainerChoiceIndex < m_Containers.Length`

Reply:
OK for now